src/components/results/dynamicsimulation/dynamic-simulation-result-tab.js

```diff
@@ -5,104 +5,78 @@
-                )
-                    .then((newlyLoadedTimeSeries) => {
-                        // insert one by one newly loaded timeserie into the cache
-                        for (const newSeries of newlyLoadedTimeSeries) {
-                            dynamicSimulationResult.timeseries.splice(
-                                dynamicSimulationResult.timeseriesMetadatas.findIndex(
-                                    (elem) =>
-                                        elem.name === newSeries.metadata.name
-                                ),
-                                1,
-                                newSeries
-                            );
-                        }
-
-                        // return selected series in newly updated cache
-                        return selectedIndexes.map((indexValue) => ({
-                            ...dynamicSimulationResult.timeseries[indexValue],
-                            index: indexValue, // memorize index position in the series names list to generate a color later in plot
-                        }));
-                    })
-                    .catch((error) => {
-                        snackError({
-                            messageTxt: error.message,
-                            headerId: 'DynamicSimulationResults',
-                        });
-                    });
-            }
-        },
-        [studyUuid, nodeUuid, dynamicSimulationResult, snackError]
+    const dynamicSimulationStatus = useSelector(
+        (state) => state.computingStatus[ComputingType.DYNAMIC_SIMULATION]
     );
 
+    const dynamicSimulationResultPresent =
+        dynamicSimulationStatus === RunningStatus.SUCCEED ||
+        dynamicSimulationStatus === RunningStatus.FAILED;
+    const handleTabChange = (_, newTabIndex) => {
+        setTabIndex(newTabIndex);
+    };
+
     return (
-        <WaitingLoader message={loadingMessage} loading={isWaiting}>
-            <DynamicSimulationResult
-                result={dynamicSimulationResult}
-                loadTimeSeries={loadTimeSeries}
-            />
-        </WaitingLoader>
+        <>
+            <Box>
+                <Tabs value={tabIndex} onChange={handleTabChange}>
+                    <Tab
+                        label={intl.formatMessage({
+                            id: TAB_INDEX_TIME_SERIES,
+                        })}
+                        value={TAB_INDEX_TIME_SERIES}
+                    />
+                    <Tab
+                        label={intl.formatMessage({
+                            id: TAB_INDEX_STATUS,
+                        })}
+                        value={TAB_INDEX_STATUS}
+                    />
+                </Tabs>
+            </Box>
+            <Box>
+                <TabPanelLazy
+                    key={TAB_INDEX_TIME_SERIES}
+                    selected={tabIndex === TAB_INDEX_TIME_SERIES}
+                >
+                    {dynamicSimulationResultPresent && (
```

Comment:
you are hiding child component when status is not "dynamicSimulationStatus === RunningStatus.SUCCEED || dynamicSimulationStatus === RunningStatus.FAILED"

does that mean the whole table is not displayed when calculation is not done yet ? does that mean we don't have the load bar as expected ?

Reply:
I think in other calculation, there is a "no result" message when no calculation has been executed yet

Reply:
YES! It is better to show always the component with a 'no result' message... This enhancement is out of the scope of this PR.

This should be done in another PR!

Reply:
DONE! Fixed in the next PR: https://github.com/gridsuite/gridstudy-app/pull/1942/files#diff-fc3137332f10d66cc215d6720cb0614aa615fb4ead30f4f573713fa0906bf2c0R130-R135